homeassistant/components/sma/sensor.py

```diff
@@ -108,7 +109,7 @@ def unique_id(self) -> str:
             f"{self._config_entry_unique_id}-{self._sensor.key}_{self._sensor.key_idx}"
         )
 
-    @property
+    @cached_property
```

Comment:
This one seems to be dynamic

Reply:
https://github.com/home-assistant/core/pull/100222